curvefs/src/metaserver/metastore.cpp

```diff
@@ -582,6 +572,12 @@ MetaStatusCode MetaStoreImpl::SendS3ChunkInfoByStream(
 }
 
 std::shared_ptr<Partition> MetaStoreImpl::GetPartition(uint32_t partitionId) {
+    ReadLockGuard readLockGuard(rwLock_);
```

Comment:
I think the modification might not improve some performance because we had to acquire the lock in both cases.

But current changes really affect the lock scope, in the previous version, load/save will hold a write lock, so there will not exist any other update or get requests. And in current version, you can't guarantee that.

Reply:
In current version, the load and save alse get write lock and the read lock acquired in the 'GetPartition' function instead of everywhere in code.